src/pydolphinscheduler/tasks/http.py

```diff
@@ -51,7 +53,29 @@ class HttpCheckCondition:
 
 
 class Http(Task):
-    """Task HTTP object, declare behavior for HTTP task to dolphinscheduler."""
+    """Task HTTP object, declare behavior for HTTP task to dolphinscheduler.
+    Attributes:
+        _task_custom_attr (set): A set containing custom attributes specific to the Http task, 
+                                including 'url', 'http_method', 'http_params', and more.
+                                
+    Args:
+        name (str): The name or identifier for the HTTP task.
+        url (str): The URL endpoint for the HTTP request.
+        http_method (str, optional): The HTTP method for the request (GET, POST, etc.). Defaults to HttpMethod.GET.
+        http_params (str, optional): Parameters for the HTTP request. Defaults to None.
+        http_check_condition (str, optional): Condition for checking the HTTP response status. 
+                                               Defaults to HttpCheckCondition.STATUS_CODE_DEFAULT.
+        condition (str, optional): Additional condition to evaluate if `http_check_condition` is not STATUS_CODE_DEFAULT.
+        connect_timeout (int, optional): Connection timeout for the HTTP request in milliseconds. Defaults to 60000.
+        socket_timeout (int, optional): Socket timeout for the HTTP request in milliseconds. Defaults to 60000.
```

Comment:
can we change the format to

```py
    :param name: A unique, meaningful string for the shell task.
    :param command: One or more command want to run in this task.
```

just like we did in shell task, in this format our docs template can auto fill the parameter type for our users